mysql-schema/mysql-schema.md

```diff
@@ -117,6 +117,7 @@ Currently, the `help_topic` is NULL.
 ## System tables related to backup and restore
 
 * `tidb_pitr_id_map`: the ID mapping information for point-in-time recovery (PITR) operations
+* `tidb_restore_registry` (New in v9.0.0): the registry for restore tasks with a unique restore ID assigned to each task to prevent conflicts between different restore operations
```

Comment:
```suggestion
* <span class="version-mark">New in v9.0.0</span> `tidb_restore_registry`: the registry for restore tasks with a unique restore ID assigned to each task to prevent conflicts between different restore operations
```